code/modules/jobs/job_types/legion.dm

```diff
@@ -97,23 +99,64 @@
 		var/obj/effect/proc_holder/spell/terrifying_presence/S = new /obj/effect/proc_holder/spell/terrifying_presence
 		H.mind.AddSpell(S)
 
-/datum/outfit/job/CaesarsLegion/Legionnaire/f13legate
+/datum/outfit/job/CaesarsLegion/Legionnaire/f13legate		// 14mm Pistol, Powerfist
 	name =			"Legate"
 	jobtype =		/datum/job/CaesarsLegion/Legionnaire/f13legate
 	shoes = 		/obj/item/clothing/shoes/f13/military/legate
 	suit = 			/obj/item/clothing/suit/armor/f13/legion/legate
 	head = 			/obj/item/clothing/head/helmet/f13/legion/legate
 	gloves =		/obj/item/clothing/gloves/legion/legate
 	glasses = 		/obj/item/clothing/glasses/sunglasses/big
-	suit_store = 	/obj/item/gun/ballistic/automatic/pistol/automag
+	suit_store = 	/obj/item/gun/ballistic/automatic/pistol/pistol14
 	r_pocket =      /obj/item/storage/bag/money/small/legion
 	l_pocket = 		/obj/item/flashlight/lantern
-	r_hand = 		/obj/item/gun/ballistic/revolver/ballisticfist
-	l_hand = 		/obj/item/ammo_box/shotgun/slug
+	r_hand = 		/obj/item/melee/powerfist/goliath
+	l_hand = 		/obj/item/tank/internals/oxygen
 	backpack = 		null
 	satchel = 		null
 	box = 			/obj/item/storage/box/legate
 
+/*  WIP Role, will be done soon
+// PRAETORIAN
+
+/datum/job/CaesarsLegion/Legionnaire/f13legate
+	title = "Praetorian"
+	flag = F13LEGATE
+	head_announce = list("Security")
+	supervisors = "Your Legate"
+	selection_color = "#ffdddd"
+	req_admin_notify = 1
+	total_positions = 0
+	spawn_positions = 0
+	outfit = /datum/outfit/job/CaesarsLegion/Legionnaire/f13legate
+	access = list()
+	minimal_access = list()
+
+/datum/outfit/job/CaesarsLegion/Legionnaire/f13legate/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_HARD_YARDS, src)
+	ADD_TRAIT(H, TRAIT_LIFEGIVER, src)
+	ADD_TRAIT(H, TRAIT_IRONFIST, src)
+	ADD_TRAIT(H, TRAIT_BIG_LEAGUES, src)
+
+/datum/outfit/job/CaesarsLegion/Legionnaire/f13praetorian
+	name =			"Praetorian"
+	jobtype =		/datum/job/CaesarsLegion/Legionnaire/f13praetorian
+	shoes = 		/obj/item/clothing/shoes/f13/military/legate
+	suit = 			/obj/item/clothing/suit/armor/f13/legion/legate
+	head = 			null
+	gloves =		/obj/item/clothing/gloves/legion/legate
+	glasses = 		/obj/item/clothing/glasses/sunglasses/big
+	suit_store = 	/obj/item/gun/ballistic/automatic/pistol/automag
+	r_pocket =      /obj/item/storage/bag/money/small/legion
+	l_pocket = 		/obj/item/flashlight/lantern
+	r_hand = 		/obj/item/gun/ballistic/revolver/ballisticfist
+	l_hand = 		/obj/item/ammo_box/shotgun/slug
+*/
```

Comment:
```suggestion
```
Avoid adding commented out code.
If it's WIP keep it on a local branch until it's complete.